sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ServiceAPITest.groovy

```diff
@@ -148,6 +149,32 @@ class ServiceAPITest extends APISpec {
         containers.each { container -> container.delete() }
     }
 
+    def "List deleted"() {
```

Comment:
Do we have these two test cases? I didn't find them. 
1. list containers including deleted ones using sas token as credential
2. undelete containers using sas token as credential
Though it's likely that those two cases are not working, at least I haven't figured out why they are not working in Python 😀

Reply:
I also don't have those test cases .NET

Reply:
@xiafu-msft We didn't add any new permission to SAS tokens for that. So adding test cases for that doesn't seem to bring additional coverage as we already test "write" permission elsewhere.
If that doesn't work then it's server side bug I think.